### ISSUE TITLE
Option to always show current protection level of tiles

### ISSUE THREAD

**Issue by cyberbeat** [Sesu8642/FeudalTactics#49]
It would be nice to have an option to always show the current protection level of tiles (perhaps even of enemys), than it would not be necessary to click on a unit. And even, when clicking on a unit, the interface currrently does not take into account the protection, for the selected unit itself. I am also often confused, when I clicked on a unit, that then the tile is empty, and I forgot where exactly it stands - it would be better to let it stay there but give it 50% opacity (or something similar).

Another wish: store last X seeds in a list, so it would be possible to play again a previous map (especially when not won).

Btw: great game! Don't add too much new features (or at least make them optional), because it impresses by its simplicity (the learning curve is not too steep).

**Comment by Sesu8642** [comment]
Thanks for your feedback and suggestions!

Since there is the options menu now, it does make sense to have such an option.

I think that it makes some sense to assume the picked up unit is no longer protecting since the player probably picked it up to move it somewhere else. It is only confusing if you don't in fact want to move that unit. The root cause seems to be that you need to pick up a unit to show the protection levels which is indeed rather unintuitive. But at least every player stumbles on this information while playing the game and no additional UI is needed. But I'm open to ideas on how to improve this.

**Comment by Sesu8642** [comment]
I agree that the game mechanics should stay close to the original. They are after all what inspired me to make this game :)
